src/components/step-sequence/step-sequence.stories.tsx

```diff
@@ -0,0 +1,216 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+
+import { StepSequence, StepSequenceItem } from ".";
+import Box from "../box";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+export const DefaultStory: ComponentStory<typeof StepSequence> = () => (
+  <StepSequence>
+    <StepSequenceItem
+      aria-label="Step 1 of 5"
+      hiddenCompleteLabel="Complete"
+      hiddenCurrentLabel="Current"
+      indicator="1"
+      status="complete"
+    >
+      Name
+    </StepSequenceItem>
+    <StepSequenceItem
+      aria-label="Step 2 of 5"
+      hiddenCompleteLabel="Complete"
+      hiddenCurrentLabel="Current"
+      indicator="2"
+      status="complete"
+    >
+      Delivery Address
+    </StepSequenceItem>
+    <StepSequenceItem
+      aria-label="Step 3 of 5"
+      hiddenCompleteLabel="Complete"
+      hiddenCurrentLabel="Current"
+      indicator="3"
+      status="current"
+    >
+      Delivery Details
+    </StepSequenceItem>
+    <StepSequenceItem
+      aria-label="Step 4 of 5"
+      hiddenCompleteLabel="Complete"
+      hiddenCurrentLabel="Current"
+      indicator="4"
+      status="incomplete"
+    >
+      Payment
+    </StepSequenceItem>
+    <StepSequenceItem
+      aria-label="Step 5 of 5"
+      hiddenCompleteLabel="Complete"
+      hiddenCurrentLabel="Current"
+      indicator="5"
+      status="incomplete"
+    >
+      Confirm
+    </StepSequenceItem>
+  </StepSequence>
+);
+
+export const Vertical: ComponentStory<typeof StepSequence> = () => (
+  <Box height="600px">
```

Comment:
What do you think of moving the `Box` to a decorator for this story so it doesn't shown in the code snippet? Happy to accept otherwise as this is minor

Reply:
I don't think it's that big of an issue they will need to render it in a Box or container and the point is to show that is flexes to the height of the container etc